### ISSUE TITLE
Support multiple metric values per aggregation pipeline

### ISSUE THREAD

**Issue by johnnyshields** [raffis/mongodb-query-exporter#24]
I'd like to be able to extract multiple metrics per aggregation pipeline. This would be more efficient because we could run one pipeline for N metrics (rather than N pipelines for N metrics).

Here's an example config structure that could be used:

```
  metrics:
  - exported_metrics:
     -  name: jobs_count_per_queue
        type: gauge
        help: "Count of jobs per queue."
        value: count
        labels: ["queue_name"]
        constLabels: []
     -  name: jobs_max_age_per_queue
        type: gauge
        help: "Maximum age of jobs per queue."
        value: max_seconds_in_queue
        labels: ["queue_name"]
        constLabels: []
     -  name: jobs_avg_age_per_queue
        type: gauge
        help: "Average age of jobs per queue."
        value: avg_seconds_in_queue
        labels: ["queue_name"]
        constLabels: []
    mode: pull
    cache: 0
    database: mongo_production
    collection: jobs
    pipeline: |
      [
        ....
        }, {
          "$project": {
            "_id": 0,
            "queue_name": "$_id",
            "count": "$count",
            "max_seconds_in_queue": "$max_seconds_in_queue",
            "avg_seconds_in_queue": "$ave_seconds_in_queue"
          }
        }
      ]
```

**Comment by raffis** [comment]
Thanks for the feature request.

Released as part of v1.0.0.
See readme.md, you need to migrate to config version v3.0.

**Comment by johnnyshields** [comment]
@raffis Thank you very much for this! Should significantly reduce load on our DB.
